config.env.py

```diff
@@ -29,3 +29,7 @@
 # Google moment
 PREFERRED_URL_SCHEME = env.get('PREFERRED_URL_SCHEME',
                                'https' if 'csh.rit.edu' in SERVER_NAME else 'http')
+
+PINGS_JOIN_ROUTE = env.get("PINGS_JOIN_ROUTE", "")
+PINGS_LEAVE_ROUTE = env.get("PINGS_LEAVE_ROUTE", "")
```

Comment:
Since these are actually UUID secrets and not full routes, it may make more sense to call these `PINGS_JOIN_UUID` and `PINGS_LEAVE_UUID` or similar